examples/plot_synthetic.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
You can leave the Microsoft part out and just make it
```
# Copyright (c) Fairlearn contributors.
```
That's basically just an add-on we used in older files before the project became independent 🙂 This applies to all files, of course